src/main/java/seedu/address/logic/commands/LocateCommand.java

```diff
@@ -121,9 +122,7 @@ public String getTemplate() {
     }
 
     @Override
-    public int getCaretIndex() {
-        return getTemplate().length();
-    }
+    public int getCaretIndex() { return getTemplate().length(); }
```

Comment:
Hey @zhangriqi, should this be refactored?

The original code follows the convention. Could you remove this change?